packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -104,6 +104,45 @@ void main() {
       );
     });
 
+    testWidgets('background color default value', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        CupertinoApp(
+          home: CupertinoTimerPicker(
+            onTimerDurationChanged: (_) { },
+          ),
+        ),
+      );
+
+      final Iterable<CupertinoPicker> pickers = tester.allWidgets.whereType<CupertinoPicker>();
+      expect(pickers.any((CupertinoPicker picker) => picker.backgroundColor != CupertinoColors.white), false);
+    });
+
+    testWidgets('background color is not null', (WidgetTester tester) async {
+      expect(
+        () {
+          CupertinoTimerPicker(
+            onTimerDurationChanged: (_) { },
+            backgroundColor: null,
+          );
+        },
+        throwsAssertionError,
+      );
+    });
+
+    testWidgets('specified background color is applied', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        CupertinoApp(
+          home: CupertinoTimerPicker(
+            onTimerDurationChanged: (_) { },
+            backgroundColor: CupertinoColors.black,
+          ),
+        ),
+      );
+
+      final Iterable<CupertinoPicker> pickers = tester.allWidgets.whereType<CupertinoPicker>();
+      expect(pickers.any((CupertinoPicker picker) => picker.backgroundColor == CupertinoColors.black), true);
```

Comment:
This check can be stricter. 
```suggestion
      expect(pickers.any((CupertinoPicker picker) => picker.backgroundColor != CupertinoColors.black), false);
```

Reply:
My Test case says "specified background color is applied" if it does not then throw the error. So I wrote in that way.